katas/args/richard/constructors.test.js

```diff
@@ -0,0 +1,78 @@
+import { declareSchema } from './constructors.js';
+import { constructLogging } from './constructors.js';
+import { constructPort } from './constructors.js';
+import { constructDirection } from './constructors.js';
+import { constructStringArray } from './constructors.js';
+import { constructIntegerArray } from './constructors.js';
+
+test('declareSchema should be retorned a json structure', () => {
+    const testSchema = {
+        symbol: '-l',
+        name: 'logging',
+        type: 'boolean',
+        value: false
+    };
+    const result = declareSchema();
+    expect(result[0]).toStrictEqual(testSchema);
+});
+
+test('constructLogging should be retorned a json structure with a logging dates', () => {
```

Comment:
check for typos: `retorned` -> `returned`. Review this in all tests, looks like it was replicated in different places

Reply:
changed :white_check_mark: